<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: rgb(255, 215, 179);}

#play_button:hover {background-color: rgb(167, 29, 34);}</string>
  </property>
  <widget class="QPushButton" name="statistics_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 221, 188);
border-radius: 10px;
border: 3px solid;
border-color: rgb(167, 29, 34);
}

QPushButton:hover:!pressed{
border: 6px solid;
border-color: rgb(255, 132, 32);
}</string>
   </property>
   <property name="text">
    <string>      Statistics</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 221, 188);
border-radius: 10px;
border: 3px solid;
border-color: rgb(166, 30, 34);
}
QPushButton:hover:!pressed{
border: 6px solid;
border-color: rgb(255, 132, 32);
}



</string>
   </property>
   <property name="text">
    <string>         New Game</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ResourceFiles.qrc">:/Images/Download Hello Hand Sign Language Concept for free.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgb(255, 221, 188);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ResourceFiles.qrc">:/Icons/fight-svgrepo-com.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 221, 188);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ResourceFiles.qrc">:/Icons/increase-svgrepo-com.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="log_out_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 221, 188);
border-radius: 10px;
border: 3px solid;
border-color: rgb(167, 29, 34);
}

QPushButton:hover:!pressed{
border: 6px solid;
border-color: rgb(255, 132, 32);
}</string>
   </property>
   <property name="text">
    <string>      Log out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ResourceFiles.qrc">:/Icons/logout-svgrepo-com.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>statistics_button</zorder>
  <zorder>play_button</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>log_out_btn</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources>
  <include location="ResourceFiles.qrc"/>
 </resources>
 <connections/>
</ui>
